spec/unit/hanami/events/adapter/memory_spec.rb

```diff
@@ -43,13 +43,9 @@
       before do
         $comment_array = []
 
-        adapter.subscribe('comment.created') do |payload|
-          $comment_array << payload
-        end
-
-        adapter.subscribe('comment.created') do |payload|
-          $comment_array << payload
-        end
+        handler = Proc.new { |payload| $comment_array << payload }
```

Comment:
I don't like this changes. We need to use a simple code in tests without any DRY. That's why `handler` block looks like an unnecessary refactoring 😞